packages/mui-system/src/cssVars/createCssVarsProvider.js

```diff
@@ -141,14 +141,19 @@ export default function createCssVarsProvider(options) {
 
     React.useEffect(() => {
```

Comment:
Aside: Pretty sure this should be a layout effect considering it does affect layout. @hbjORbj

Reply:
Oh, I see. Will make the change!